automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -5,236 +5,265 @@ import java.util.UUID
-          detailPage.share(Config.Users.ron.email, "READER", true)
-          detailPage.signOut()
-          val listPage2 = signIn(Config.Users.ron)
-          val detailPage2 = listPage2.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
-          detailPage2.ui.hasShareButton shouldBe true
+          withSignIn(Config.Users.draco) { listPage =>
+
+            val detailPage = listPage.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
+            detailPage.share(Config.Users.ron.email, "READER", true)
+            detailPage.signOut()
+            val listPage2 = signIn(Config.Users.ron)
+            val detailPage2 = listPage2.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
+            detailPage2.ui.hasShareButton shouldBe true
+          }
         }
+
       }
 
-      "should be able to set can compute permissions for users that are writers" in withWebDriver {implicit driver =>
+      "should be able to set can compute permissions for users that are writers" taggedAs Tags.GooglePassing in withWebDriver {implicit driver =>
         withWorkspace(billingProject, "WorkspaceSpec_share") { workspaceName =>
-          val listPage = signIn(Config.Users.harry)
-          val detailPage = listPage.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
-          val aclEditor = detailPage.openShareDialog(Config.Users.ron.email, "WRITER")
-          aclEditor.ui.canComputeEnabled() shouldBe true
-          aclEditor.ui.canComputeChecked() shouldBe false
+          withSignIn(Config.Users.draco) { listPage =>
+            val detailPage = listPage.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
+            val aclEditor = detailPage.openShareDialog(Config.Users.ron.email, "WRITER")
+            aclEditor.ui.canComputeEnabled() shouldBe true
+            aclEditor.ui.canComputeChecked() shouldBe false
+          }
         }
+
       }
 
       "should not be able to set/change can compute permissions for other owners" in withWebDriver {implicit driver =>
         withWorkspace(billingProject, "WorkspaceSpec_share") { workspaceName =>
-          api.importMetaData(billingProject, workspaceName, "entities", TestData.SingleParticipant.participantEntity)
-          api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, SimpleMethod, SimpleMethodConfig.configNamespace, s"$methodConfigName Config", 1,
-            SimpleMethodConfig.inputs, SimpleMethodConfig.outputs, "participant")
-          val listPage = signIn(Config.Users.harry)
-          val detailPage = listPage.openWorkspaceDetails(billingProject, workspaceName).awaitLoaded()
-          val aclEditor = detailPage.openShareDialog(Config.Users.ron.email, "OWNER")
-          aclEditor.ui.canComputeEnabled() shouldBe false
-          aclEditor.ui.canComputeChecked() shouldBe true
+          withSignIn(Config.Users.draco) { listPage =>
```

Comment:
FYI, the `with*` functions do not necessarily need to be invoked at the very beginning of the test case. After the `in` of the spec, we're just in regular Scala land, not in the spec DSL. See feedback about `SignInSpec`.